src/main/webapp/js/Registration.js

```diff
@@ -0,0 +1,105 @@
+const React = require('react');
+const TITLE_HEIGHT = 100;
```

Comment:
Is there a way to move this to CSS and use css-classes to retrieve these values?